Source/test-utils.js

```diff
@@ -56,7 +56,8 @@ export function createSharedStyle(document, Primitive, style) {
 }
 
 export function outputPath() {
-  const output = `${NSTemporaryDirectory()}SketchAPI/`
+  const uuid = NSUUID.UUID().UUIDString()
+  const output = NSTemporaryDirectory().stringByAppendingPathComponent('SketchAPI/').stringByAppendingPathComponent(uuid).stringByAppendingPathComponent('/')
   fs.mkdirSync(output, { recursive: true })
   return output
```

Comment:
Given that we're using `outputPath/…` everywhere I think we can drop the last appended `/`. Then double check we _are_ actually using `/` in the places where we use the path since the topic has come up a few times recently. Maybe we can improve readability a little with some line breaks, too?

```suggestion
  const uuid = NSUUID.UUID().UUIDString()
  const path = NSTemporaryDirectory()
    .stringByAppendingPathComponent('SketchAPI/')
    .stringByAppendingPathComponent(uuid)
  
  fs.mkdirSync(path, { recursive: true })
  return path
```